src/main/java/org/bookstore/dto/book/BookDtoWithoutCategoryIds.java

```diff
@@ -0,0 +1,7 @@
+package org.bookstore.dto.book;
+
+import java.math.BigDecimal;
+
+public record BookDtoWithoutCategoryIds(Long id, String title, String author, String isbn,
+                                        BigDecimal price, String description, String coverImage) {
```

Comment:
```suggestion
public record BookDtoWithoutCategoryIds(Long id, 
                                                                         String title, 
                                                                         String author, 
                                                                         String isbn,
                                                                          BigDecimal price, 
                                                                          String description, 
                                                                          String coverImage ) {
```
Let’s  apply this style for each dto